public/index.html

```diff
@@ -12,7 +12,7 @@
     -->
 
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
-    <link rel="stylesheet" type="text/css" href="https://cdn.fromdoppler.com/doppler-ui-library/v2.1.0/css/styles.css">
+    <link rel="stylesheet" type="text/css" href="https://cdn.fromdoppler.com/doppler-ui-library/build333/css/styles.css">
```

Comment:
maybe its better don commit this file until have final version dont?

Reply:
Ok!

Reply:
Committing it is useful to allow us to test it online.

We could add a rule in some place (I do not know where and how) to do not allow merge without definitive versions.